contracts/TokenForwarding.cdc

```diff
@@ -0,0 +1,49 @@
+/**
+
+# Non-Fungible Token Forwarding Contract
+
+*/
+
+import NonFungibleToken from 0x02
+
+pub contract NFTForwarding {
+
+    // Event that is emitted when NFTs are forwarded to the target receiver
+    pub event ForwardedToken(id: UInt64, from: Address?)
+
```

Comment:
Maybe it would be good to include a canonical NFT forwarder path that could be used when storing the forwarder? something like this
```suggestion
let nftForwarderPath: StoragePath
```

Reply:
could you share why this makes sense? in this case it just gets defined in the creation transaction, and never really gets used again in the actual contract. Is this just to expose the path to future references of the forwarder?

Reply:
Yeah, whenever someone wants to add a forwarder to their account, they can use the contract-defined path instead of having to hard code it themselves